x/expiration/spec/03_messages.md

```diff
@@ -0,0 +1,88 @@
+# Expiration Messages
+
+In this section we describe the processing of the expiration messages and the corresponding updates to the state.
+All created/modified state objects specified by each message are defined within the [state](02_state.md) section.
+
+<!-- TOC -->
+  - [Entries](#entries)
+    - [Msg/AddExpirationRequest](#msg-addexpirationrequest)
+    - [Msg/ExtendExpirationRequest](#msg-extendexpirationrequest)
+    - [Msg/InvokeExpirationRequest](#msg-invokeexpirationrequest)
+  - [Authz Grants](#authz-grants)
+
+---
+## Entries
+
+### Msg/AddExpirationRequest
+
+An expiration is created using the `AddExpiration` service method.
+
+Expirations are identified using their `module_asset_id`.
+
+#### Request
+
++++ https://github.com/provenance-io/provenance/blob/eb569b71b4d9137272432df5968cd62bf1eca2fb/proto/provenance/expiration/v1/tx.proto#L22-L32
```

Comment:
This and others like it currently point to the `epic/expiration` branch. Will these need to point to main once merged?